xbmc/pvr/guilib/PVRGUIActions.cpp

```diff
@@ -1219,11 +1221,23 @@ namespace PVR
       CPVRStreamProperties props;
 
       if (item->IsPVRChannel())
-        client->GetChannelStreamProperties(item->GetPVRChannelInfoTag(), props);
+      {
+        if (epgProps) // we already have props from PlayEpgTag()
+          props = *epgProps;
```

Comment:
If it correct to use EPG stream props for a channel?

Reply:
This channel was created in PlayEpgTag(), so in this case yes.

Would you prefer it was renamed to something more generic?

Reply:
Naming the var like what it actually represents is just fine. However, "if channel use epg props" reads somehow wrong/suspicious. We probably should add a comment explaining that this is intentional and why.

Reply:
Makes sense.

Reply:
Ok, updated comment